handlers/missing_test/__init__.py

```diff
@@ -9,6 +9,8 @@
 class MissingTestHandler(EventHandler):
     COMPONENT_DIRS_TO_CHECK = ('layout', 'script', 'gfx', 'style', 'net')
     TEST_DIRS_TO_CHECK = ('ref', 'wpt', 'unit')
+    TEST_FILES_TO_CHECK = ('components/script/dom/testbinding.rs',
+                           'components/script/dom/webidls/TestBinding.webidl')
```

Comment:
Let's also add TestBindingProxy.webidl, TestBindingIterable.webidl, TestBindingPairIterable.webidl, testbindingproxy.rs, testbindingiterable.rs, and testbindingpairiterable.rs.